### ISSUE TITLE
iOS accessibility 

### ISSUE THREAD

**Issue by KeeJef** [loki-project/session-ios#193]
Received a report about the lack of Accessibility of Session for Blind/Deaf users 

> Sadly, Signal, the chat application you based Session on, already has the following accessibility issues:
> - The "move to last message" button is unlabeled for the blind. When focus moves to that button, VoiceOver, the apple-integrated screen reader, only announces "button"—and that's what gets displayed on braille displays as well. Session inherited this issue.
> - The attachment menu is accessible, but ALL submenus are NOT. Certain actions can get performed, but ultimately sending an attachment is impossible. Also, many icons have "bad" names: "ic flash mode off button", for instance, or "X button" to close a submenu. Session inherits these problems.
> - Actions such as reply, forward etc. are not accessible for the blind. Session probably inherits these problems, provided it implements these actions.
> - Media in chat histories are not accessible, nor are media-related actions such as safe or forward. Session probably inherits these problems.
> 
> Additionally, Session has the following accessibility-related issues:
> - Focus is trapped on the chat history. I cannot access settings, the box to send session requests, or anything other than already existing chats. The only way I can start a action with someone else is if they reach out to me first, and even in that case I have to write my session ID down somewhere, because I can't access that in app, either.
> - The "submit" button is unlabeled if the text input for messages is empty, and turns to "UpArrowDarkMode button" when there's text in the text input.
> - Options to choose certain things, like which notification-type should get used, probably only use a visual selection indicator. Consider using native radiobuttons/checkboxes or related equivalents, or at least set the accessibility roles and attributes appropriately.
>

**Comment by nielsandriesse** [comment]
As of version 1.7.3 we properly support VoiceOver. We'll be improving things further in the future, but for now I'm closing this issue.
